monitor/api/pom.xml

```diff
@@ -20,13 +20,320 @@
+      		<artifactId>brooklyn-rest-client</artifactId>
+    	</dependency> 
+    	<!--<dependency>
+    		<groupId>io.brooklyn</groupId>
+    		<artifactId>brooklyn-rest-client</artifactId>
+    		<version>0.7.0-M1</version>
+		</dependency>-->
+
+		<dependency>
+    		<groupId>com.sun.jersey</groupId>
+    		<artifactId>jersey-server</artifactId>
+    		<version>1.19</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.sun.jersey</groupId>
+			<artifactId>jersey-bundle</artifactId>
+			<version>1.19</version>
+		</dependency>
+
+		<dependency>
+    		<groupId>com.sun.jersey</groupId>
+    		<artifactId>jersey-core</artifactId>
+    		<version>1.19</version>
+		</dependency>
+
+		<dependency>
+    		<groupId>com.sun.jersey</groupId>
+    		<artifactId>jersey-servlet</artifactId>
+    		<version>1.19</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.sun.jersey</groupId>
+			<artifactId>jersey-client</artifactId>
+			<version>1.19</version>
+		</dependency>
+
+        <dependency>
+         	<groupId>eu.seaclouds-project</groupId>
+         	<artifactId>monitor-core</artifactId>
+         	<version>0.1.0-SNAPSHOT</version>
+         	<scope>system</scope>
+         	<systemPath>${basedir}/../core/target/monitor-core-0.1.0-SNAPSHOT.jar</systemPath>
+      	</dependency>
+
+		<dependency>
+			<groupId>it.polimi.modaclouds.monitoring</groupId>
+			<artifactId>metrics-observer</artifactId>
+			<version>3.0.1</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>org.restlet</artifactId>
+					<groupId>org.restlet.jse</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>rsp-services-api</artifactId>
+					<groupId>polimi.deib</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>qos-models</artifactId>
+					<groupId>it.polimi.modaclouds.qos-models</groupId>
+				</exclusion>
+				<exclusion>
+					<artifactId>knowledge-base-api</artifactId>
+					<groupId>it.polimi.modaclouds.monitoring.knowledge-base-api</groupId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+	</dependencies>
+
+
+	<build>
+		<plugins>
+			<plugin>
+  				<artifactId>maven-war-plugin</artifactId>
+  				<version>2.6</version>
+  				<configuration>
+    				<webXml>WebContent\WEB-INF\web.xml</webXml>        
+  				</configuration>
+			</plugin>
+
+			<plugin>
+        		<artifactId>maven-antrun-plugin</artifactId>
+        		<version>1.6</version>
+        		<executions>
+          			<execution>
+            			<phase>generate-resources</phase>
+            			<configuration>
+              			<tasks>
+              				<mkdir dir="${basedir}/lib/"/>
+                			<mkdir dir="${basedir}/WebContent/WEB-INF/lib/"/>
+              			</tasks>
+            			</configuration>
+            			<goals>
+              				<goal>run</goal>
+            			</goals>
+          			</execution>
+        		</executions>
+      		</plugin>
+
+			<plugin>
+    			<artifactId>maven-antrun-plugin</artifactId>
+    			<version>1.6</version>
```

Comment:
I would only save the `antrun` tasks for the 2 conflicting libraries (see https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/41/files#diff-344e50cefa0780cf204498bf7450e6ecR78)  which don't have Maven support (as a temporal solution). I have the impression that the rest of the libraries being downloaded can be found in a public Maven repository. Please convert them to Maven dependencies if possible.